apps/agreement/package.json

```diff
@@ -13,21 +13,22 @@
   "scripts": {
     "console": "buidler console",
     "compile": "buidler compile --force",
+    "compile:staking": "buidler compile --config buidler.config.staking.js",
     "devchain": "buidler node",
     "build": "cd app && yarn build",
     "lint": "solium --dir ./contracts",
-    "test": "buidler test",
-    "test:gas": "REPORT_GAS=true buidler test --network localhost",
+    "test": "yarn compile:staking && buidler test",
+    "test:gas": "REPORT_GAS=true yarn test --network localhost",
     "coverage": "buidler coverage --network coverage",
     "abi:extract": "buidler-extract --output abi/ --keys abi"
   },
   "dependencies": {
-    "@aragon/os": "5.0.0-beta.8"
+    "@aragon/minime": "1.0.0",
```

Comment:
Can we move minime back to a devDependency?

Reply:
Definitely!